user-guide/Advanced_Functionality/Databases/Configuring_dedicated_clustered_storage/Indexing_database/OpenSearch_Database/Installing_OpenSearch_database.md

```diff
@@ -21,13 +24,348 @@ See the [official documentation](https://opensearch.org/docs/latest/) on how to
+
+Configure the *OpenSearch.yml* file as illustrated in the example below.
+
+```yml
+plugins.security.disabled: false
+
+#Transport layer TLS
+plugins.security.ssl.transport.keystore_type: PKCS12
+plugins.security.ssl.transport.keystore_filepath: /etc/opensearch/cert/FQDNOfYourNode-node-keystore.p12
+plugins.security.ssl.transport.keystore_password: ReplaceMeByGeneratedPasswordByGithubScript
+plugins.security.ssl.transport.truststore_type: PKCS12
+plugins.security.ssl.transport.truststore_filepath: /etc/opensearch/cert/FQDNOfYourNode-node-keystore.p12
+plugins.security.ssl.transport.truststore_password: ReplaceMeByGeneratedPasswordByGithubScript
+
+#REST Layer TLS
+plugins.security.ssl.http.enabled: true
+plugins.security.ssl.http.keystore_type: PKCS12
+plugins.security.ssl.http.keystore_filepath: /etc/opensearch/cert/FQDNOfYourNode-node-keystore.p12
+plugins.security.ssl.http.keystore_password: ReplaceMeByGeneratedPasswordByGithubScript
+plugins.security.ssl.http.truststore_type: PKCS12
+plugins.security.ssl.http.truststore_filepath: /etc/opensearch/cert/FQDNOfYourNode-node-keystore.p12
+plugins.security.ssl.http.truststore_password: ReplaceMeByGeneratedPasswordByGithubScript
+
+plugins.security.allow_default_init_securityindex: true
+plugins.security.nodes_dn:
+  - 'CN=FQDNOpenSearchNode1,OU=NameOfYourCluster,O=OpenSearch,C=BE'
+  - 'CN=FQDNOpenSearchNode2,OU=NameOfYourCluster,O=OpenSearch,C=BE'
+  - 'CN=FQDNOpenSearchNode3,OU=NameOfYourCluster,O=OpenSearch,C=BE'
+plugins.security.audit.type: internal_opensearch
+plugins.security.enable_snapshot_restore_privilege: true
+plugins.security.check_snapshot_restore_write_privileges: true
+plugins.security.restapi.roles_enabled: ["all_access", "security_rest_api_access"]
+plugins.security.system_indices.enabled: true
+plugins.security.system_indices.indices: [".plugins-ml-model", ".plugins-ml-task", ".opendistro-alerting-config", ".opendistro-alerting-alert*", ".opendistro-anomaly-results*", ".opendistro-anomaly-detector*", ".opendistro-anomaly-checkpoints", ".opendistro-anomaly-detection-state", ".opendistro-reports-*", ".opensearch-notifications-*", ".opensearch-notebooks", ".opensearch-observability", ".opendistro-asynchronous-search-response*", ".replication-metadata-store",".opendistro-anomaly-detection-state", ".opendistro-reports-*", ".opensearch-notifications-*", ".opensearch-notebooks", ".opensearch-observability", ".opendistro-asynchronous-search-response*", ".replication-metadata-store"]
+node.max_local_storage_nodes: 3
+indices.query.bool.max_clause_count: 2147483647
+```
+
+##### Restart OpenSearch
+
+1. When you have finished the TLS configuration and [set a different hash for the admin user](#user-configuration), restart OpenSearch. You can use the following command for this:
+
+   ```bash
+   sudo systemctl restart opensearch
+   ```
+
+1. Check via the command line if data is returned:
+
+   ```curl
+   curl https://166.206.186.146:9200 -u admin:yournewpassword --ssl-no-revoke
```

Comment:
does the option --ssl-no-revoke work for linux?
A quick google search suggests otherwise.
In any case i have always used --insecure during testing, should maybe work for both?
Its a bit confusing if this is a windows only command as the one above here is linux only (and the whole doc is talking about installing on linux)

Reply:
The --ssl-no-revoke works for Linux. I get cluster_name etc from it.

Reply:
Okay thats good news